Modules/_testcapimodule.c

```diff
@@ -2340,6 +2340,31 @@ get_timezone_utc_capi(PyObject* self, PyObject *args) {
     }
 }
 
+static PyObject *
+get_date_fromdate(PyObject *self, PyObject *args)
+{
+    PyObject *rv = NULL;
+    int year = 0, month = 0, day = 0;
+    int macro = 0;
+
+    if (!PyArg_ParseTuple(args, "iii|p", &year, &month, &day, &macro))
+    {
```

Comment:
Formatting nitpick: [PEP 7](https://www.python.org/dev/peps/pep-0007/#code-lay-out) says that for if statements, the curly brace goes on the line with the conditional. (The curly brace after the function definition *is* on its own line).